packages/css/src/components/mega-menu/mega-menu.scss

```
@@ -0,0 +1,34 @@
+/**
+ * @license EUPL-1.2+
+ * Copyright (c) 2023 Gemeente Amsterdam
+ */
+
+@import "../../common/breakpoint";
+
+.amsterdam-mega-menu {
+  padding-block: 3.5rem 4.5rem;
+  padding-inline: 1.5rem;
+
+  @media screen and (min-width: $amsterdam-breakpoint-wide) {
+    padding-block: 5rem 7.5rem;
+    padding-inline: 5.24rem;
+  }
+}
+
+.amsterdam-mega-menu .amsterdam-grid__cell {
+  .amsterdam-heading:nth-child(n + 1) {
+    padding-block-start: 3.5rem;
+  }
+}
+
+.amsterdam-mega-menu .amsterdam-unordered-list {
```

This creates a dependency between Megamenu and UnorderedList. We could create a MegaMenu subcomponent (MegaMenu.ListWrapper or something) which sets the `columns` styling, so we could get rid of that dependency.

---

The `columns` property would be inherited by the list, when we set it on something like `MegaMenu.ListWrapper`

---

I would rather target the tag then introduce a new wrapper.
```suggestion
.amsterdam-mega-menu ul, .amsterdam-mega-menu ol {
```